File: src/class/map/tile.py
from ..util .jsonObjectConvert import JsonObjectConvert
from .wang import Wang


class Tile (JsonObjectConvert):
    def __init__(self, walkable: bool, wang: Wang):
        self.walkable = walkable
        self.wang = wang

    def __str__(self):
        return "{walkable:" + str(self.walkable) + ",wang:" + str(self.wang) + "}"

File: src/class/map/tile.ts
import { Wang } from "./wang"
export class Tile{
    walkable: Boolean;
    wang: Wang;
    
    constructor(walkable: Boolean, wang: Wang){
        this.walkable = walkable
        this.wang = wang
    }

    public toString(){
        return "{walkable:" + this.walkable + ",wang:" + this.wang + "}"
    }
}